src/main/thrift/parquet.thrift

```diff
@@ -715,6 +715,10 @@ struct DictionaryPageHeader {
  * New page format allowing reading levels without decompressing the data
  * Repetition and definition levels are uncompressed
  * The remaining section containing the data is compressed if is_compressed is true
+ *
+ * N.B. this page header is not necessarily strictly better then DataPageHeader.
+ * Page indexes already require that rows are aligned on page boundaries, and compressing
+ * repetition and definition levels can still be effective in some cases.
```

Comment:
Are you saying this is deprecated? Why do we need this comment?  It's not clear what you're trying to achieve here.  (Nit: prefer not to use abbreviations like N.B.)